api/serializers.py

```
@@ -0,0 +1,68 @@
+from diab_labs.models import LabResult
+from diab_medications.models import MedicationOrder
+from clinical_refs.models import ClinicalReference
+from ai_summarizer.models import AISummary
+
+
+class PatientSerializer(serializers.ModelSerializer):
+    primary_doctor_id = serializers.UUIDField(source='primary_doctor.id', read_only=True)
+    
+    class Meta:
+        model = Patient
+        fields = ['id', 'national_id', 'full_name', 'dob', 'sex', 'primary_doctor', 'primary_doctor_id', 'created_at']
```

Ruff (0.12.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)